wgpu-core/Cargo.toml

```diff
@@ -33,6 +33,7 @@ ignored = ["cfg_aliases"]
 
 [features]
 default = ["link"]
+counters = ["wgt/counters"]
```

Comment:
Features should be documented  - this is the right place to note down the impact of this and what it gives to a user